cpp/src/LocationRequest.cpp

```diff
@@ -307,6 +313,14 @@ LocationRequest::LocationRequest(rapidjson::Value &json) {
 		useSVD = false;
 	}
 
+	// useSVD
```

Comment:
totally minor, just a copy and paste thing, this should be for the Reassess variable rather than SVD